core/src/main/java/com/homeaway/streamplatform/streamregistry/UUIDMDCResponseFilter.java

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2018 Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamplatform.streamregistry;
+
+import static com.homeaway.streamplatform.streamregistry.UUIDMDCRequestFilter.MDC_REQUEST_UUID;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+
+import org.slf4j.MDC;
```

Comment:
added the Request UUID in the response headers so that StreamRegistryJavaClients and SMP can log the UD

```{
  "request-uuid": "bb153302-80e5-4843-9914-060570bf375d",
  "date": "Fri, 01 Mar 2019 17:20:29 GMT",
  "content-length": "0",
  "content-type": null
}```